### DISCUSSION TITLE
Adding a new location is confusing

### DISCUSSION THREAD

**Discussion by JohanG-75** [breezy-weather/breezy-weather#1145] [Ideas]
### Describe your idea

When I see a search button and a list, i expect the search button to look through the list. Please consider changing the search button with the '+' button. 
The string "search for a location..." within is confusing too, consider "add a new location..." or something more descriptive.  

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.4 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Which search button are you talking about?

**Comment by JohanG-75** [comment]
The one in the Locations panel.

**Reply by papjul** [reply]
I don't see any

**Reply by JohanG-75** [reply]
Both LineageOS 21 and the latest android have it. 
 <details>
<summary>Screenshot</summary>  

![Locations panel screenshot](https://github.com/breezy-weather/breezy-weather/assets/63257154/43e3a17d-c2df-433d-9bf4-e19e5f9fb665)
</details>

**Reply by papjul** [reply]
This is not Breezy Weather v5.2.4 unlike what you said

**Reply by JohanG-75** [reply]
Indeed, my apologies. Updated the repository and the new version does have a clearer locations tab.
